cloudinit/cmd/devel/logs.py

```diff
@@ -179,6 +232,14 @@ def collect_logs(tarfile, include_userdata: bool, verbosity=0):
                 1,
                 verbosity,
             )
+        if os.path.exists(os.path.join(CLOUDINIT_RUN_DIR, "disabled")):
```

Comment:
Is there a reason we don't always want whats in this dir?

Reply:
Only reason is that those files we care about are already symlinked into /run/cloud-init when cloud-init is enabled. So, it'd be duplication in the typical enabled case.
```bash
lrwxrwxrwx 1 root root   36 Oct 17 20:19 result.json -> ../../var/lib/cloud/data/result.json
....
lrwxrwxrwx 1 root root   36 Oct 17 20:19 status.json -> ../../var/lib/cloud/data/status.json
```